docs/devguide/safety_profile.rst

```diff
@@ -77,10 +77,10 @@ OpenDDS uses ACE as its platform abstraction library, and in OpenDDS's Safety Pr
 * FACE Safety Extended with Standard C++ Dynamic Allocation
 
 OpenDDS's configure script will automatically configure ACE.
-Pass the command line argument “--safety-profile=base” to select the Safety Base profile.
-Otherwise a “--safety-profile” (no equals sign) configuration will default to Safety Extended with Memory Pool.
+Pass the command line argument "--safety-profile=base" to select the Safety Base profile.
+Otherwise a "--safety-profile" (no equals sign) configuration will default to Safety Extended with Memory Pool.
```

Comment:
```suggestion
Pass the command line argument ``--safety-profile=base`` to select the Safety Base profile.
Otherwise a ``--safety-profile`` (no equals sign) configuration will default to Safety Extended with Memory Pool.
```